soh/soh/OTRGlobals.cpp

```diff
@@ -1676,6 +1688,22 @@ extern "C" GetItemEntry Randomizer_GetItemFromKnownCheck(RandomizerCheck randomi
     return ItemTable_RetrieveEntry(getItemModIndex, itemID);
 }
 
+extern "C" GetItemEntry Randomizer_GetItemFromKnownCheckWithoutObtainabilityCheck(RandomizerCheck randomizerCheck, GetItemID ogId) {
```

Comment:
could be cool to refactor this to something like
```c
Randomizer_GetItemFromKnownCheck(RandomizerCheck randomizerCheck, GetItemID ogId, bool checkObtainability)
```

Reply:
Yea I wanted this as well but wasn't sure if it would be as easy as using a default argument or if I'd have to through and change every place `Randomizer_GetItemFromKnownCheck` is called again

Reply:
it's still a refactor i'd like to see, but we can consider it out of scope for now. let's make a v3 issue for it

Reply:
#1492